x/treasury/client/cli/query.go

```diff
@@ -224,37 +133,17 @@ $ terracli query treasury reward-weight 14
 // GetCmdQueryTaxProceeds implements the query tax-proceeds command.
 func GetCmdQueryTaxProceeds(cdc *codec.Codec) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "tax-proceeds [epoch]",
-		Args:  cobra.RangeArgs(0, 1),
-		Short: "Query the tax proceeds for the epoch",
+		Use:   "tax-proceeds",
+		Args:  cobra.NoArgs,
+		Short: "Query the tax proceeds for the current epoch",
 		Long: strings.TrimSpace(`
-Query the tax proceeds corresponding to the given epoch. The return value will be sdk.Coins{} of all the taxes collected. 
+Query the tax proceeds corresponding to the current epoch. The return value will be sdk.Coins{} of all the taxes collected. 
```

Comment:
Do we need to let people query the tax proceeds of the current epoch? Where is this being used right now?

Reply:
FCD will use this

Reply:
Ok